lib/build.js

```diff
@@ -185,6 +185,13 @@ function build (gyp, argv, callback) {
       }
     }
 
+    if (!win) {
+      // Add build-time dependency symlinks (such as Python) to PATH
+      const buildBinsDir = path.resolve('build', 'node_gyp_bins')
+      process.env.PATH = `${buildBinsDir}:${process.env.PATH}`
+      log.verbose('bin symlinks', `adding symlinks (such as Python), at "${buildBinsDir}", to PATH`)
```

Comment:
```suggestion
      log.verbose('bin symlinks', `adding symlinks (such as Python 3), at "${buildBinsDir}", to PATH`)
```

Reply:
Feels like bikeshedding, but I don't like how specific this is. Implicitly, all Python in this version of `node-gyp` and beyond would be Python 3. There will never be an ambiguity in which major version of Python was symlinked.

Unless the Pythonistas unleash Python 4 on us, at which point heaven save us.